x11-packages/firefox/build.sh

```diff
@@ -68,6 +85,11 @@ termux_step_pre_configure() {
 	export HOST_CC=$(command -v clang)
 	export HOST_CXX=$(command -v clang++)
 
+	export BINDGEN_CFLAGS="--target=$CCTERMUX_HOST_PLATFORM --sysroot=$TERMUX_PKG_TMPDIR/firefox-toolchain/sysroot"
+	local env_name=BINDGEN_EXTRA_CLANG_ARGS_${CARGO_TARGET_NAME^^}
```

Comment:
The `^^` parameter expansion and related - `^`, `,`, `,,` - are deprecated[^1] as of Bash 5.1[^2] consider using `@U`[^3] instead.
```suggestion
	local env_name=BINDGEN_EXTRA_CLANG_ARGS_${CARGO_TARGET_NAME@U}
```

[^1]: In the same way as `let` or `$[]` for example.
[^2]: https://mywiki.wooledge.org/BashFAQ/061
[^3]: https://man7.org/linux/man-pages/man1/bash.1.html#EXPANSION see `${parameter@operator}`

Reply:
Citation needed for [1]. No where is it mentioned deprecated. Instead of saying deprecated as of bash 5.1, this requires minimum bash 5.1 (2020) for zsh inspired feature whereas the usual method works since bash 4.0 (2009).

Reply:
What do you want me to say to this?
I made this comment 3 and a half months ago and it was an unnecessary nitpick.

Our CI runs on Ubuntu 24.04 which ships Bash 5.2,
or at that point still ran on Ubuntu 22.04 (shipping Bash 5.1).

I think we shouldn't be shy to use the syntax afforded to us by the version of Bash we can reliably expect to be present.

Reply:
I am doing some clean up of rust. Still looking whether to incoporate this or not.

Reply:
I personally think 4 years is plenty old to be considered a "stable" feature.
But I don't have any strong opinions either way.